### PR TITLE
Turned all 0x1b_u8.into() into '\x1b' in the docs

### PR DESCRIPTION
**Pull request by mahmoud-moursy**
Housekeeping to keep docs clean and up to date -- turned all instances of `0x1b_u8.into()` into `\x1b` when I stumbled upon this in the Rust docs.

Tested these changes using this code:
```rs
fn main() {
    let esc: char = 0x1b_u8.into();
    let esc_2 = '\x1b';
    
    assert_eq!(esc, esc_2);
}
```
It exited and returned code `0`.

### CODE REVIEW

File: library/core/src/char/methods.rs [hunk 1]
```diff
@@ -1250,7 +1250,7 @@ impl char {
     /// let percent = '%';
     /// let space = ' ';
     /// let lf = '\n';
-    /// let esc: char = 0x1b_u8.into();
+    /// let esc: char = '\x1b';
```
Comment (r00ster91) [inline library/core/src/char/methods.rs:1253]:
```suggestion
    /// let esc = '\x1b';
```
You could now also remove all of the `char` type annotations.

**Comment by r00ster91** [comment]
There seem to be more occurrences in https://github.com/rust-lang/rust/blob/master/library/core/src/num/mod.rs like for example:
```
    /// let lf = b'\n';
    /// let esc = 0x1b_u8;
    ///
```
Do you want to turn those into `b'\x1b'`?

**Comment by T-O-R-U-S** [comment]
> Do you want to turn those into `b'\x1b'`?

I would love to, but traversing the entirety of Rust's source code to find these instances would take me a lot of time.

**Comment by r00ster91** [comment]
@T-O-R-U-S there are actually not that many. Here's what I meant: #91044

**Comment by m-ou-se** [comment]
Closing this, since #91044 has already been merged.

**Pull request closed without merging**